src/libsync/networkjobs.cpp

```diff
@@ -328,21 +337,21 @@ bool LsColJob::finished()
         connect( &parser, SIGNAL(directoryListingIterated(const QString&, const QMap<QString,QString>&)),
                  this, SIGNAL(directoryListingIterated(const QString&, const QMap<QString,QString>&)) );
         connect( &parser, SIGNAL(finishedWithError(QNetworkReply *)),
-                 this, SIGNAL(finishedWithError(QNetworkReply *)) );
+                 this, SIGNAL(parserFinishedWithError(QNetworkReply *)) );
```

Comment:
s/SIGNAL/SLOT/     parserFinishedWithError is a slot.

But actually, you can connect signal with many arguments to slot with less argument. So you can simply do:

```
  connect( &parser, SIGNAL(finishedWithError(QNetworkReply *)),
                 this, SIGNAL(finishedWithError()) );
```

inless you intended to do tomething with the QNetworkReply